pkgs/games/chessx/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, pkgconfig, pkgs, qtbase, qtsvg, qttools, qtmultimedia, fetchurl }:
+pkgs.stdenv.mkDerivation rec {
+  name = "chessx";
+  version = "1.3.2";
+  src = fetchurl {
+    url = "mirror://sourceforge/chessx/chessx-${version}.tgz";
+    sha256 = "b136cf56d37d34867cdb9538176e1703b14f61b3384885b6f100580d0af0a3ff";
+  };
+  preConfigure = ''
+    qmake -spec linux-g++ chessx.pro
+  '';
+  buildInputs = [
+   stdenv
+   pkgconfig
+
+   qtbase
+   qtsvg
+   qttools
+   qtmultimedia
+
+
+   pkgs.zlib
```

Comment:
Please take `zlib` as an argument directly and get rid of the `pkgs` argument.

Also, why all this whitespace in the list?